src/components/ChessBoard.js

```diff
@@ -6,14 +6,33 @@ function generateTemplete() {
   <style>
 
     .square {
-      width: var(--square-size);
+      width: var(--v-w);
+      min-width: calc(var(--small-size));
       aspect-ratio: 1;
       font-size: calc(var(--square-size) * .75);
       text-align: center;
       position: relative;
       display: flex;
     }
 
+    @media(min-width: 700px) {
+      .square {
+        width: var(--medium-size)
+      }
+    }
+
+    @media(min-width: 1000px) {
+      .square {
+    width: calc(var(--full-size));
```

Comment:
ditto - we can extract these and redefine them

Reply:
I am confuzzled